src/components/shared/Templates/MeetingApplyTemplate/index.tsx

```diff
@@ -1,33 +1,69 @@
+import { useState } from 'react';
+import { ChangeEventHandler } from 'react';
+import { useNavigate } from 'react-router-dom';
 import ContentBox from 'src/components/shared/ContentBox';
 import FooterButton from 'src/components/shared/FooterButton';
 import Friend from 'src/components/shared/Friend';
 import FriendWithCheck from 'src/components/shared/FriendWithCheck';
 import { PageLayout } from 'src/components/shared/Layout';
 import Search from 'src/components/shared/Search';
+import { useCreateMeetingQuery } from 'src/hooks/api/home';
 import { FriendType, MeetingTitle } from 'src/types/meeting';
 
+import Input from '../../Input';
 import $ from './style.module.scss';
 
 type Props = {
   isApply?: boolean;
   title: string;
   friendFetchData: FriendType[];
-  addedList: number[];
-  setAddedList: React.Dispatch<React.SetStateAction<number[]>>;
+  addedFriendList: number[];
+  setAddedFriendList: React.Dispatch<React.SetStateAction<number[]>>;
```

Comment:
혹시 `setAddedFriendList` 함수를 그대로 넘겨받지 말고 `(friendsID) => setAddedFriendList(friendsID)` 형태로 넘겨받는 건 어떨까요? 어떤 역할을 하는 함수인지 타입 선언부가 좀 더 명확해질 것 같습니다~
```suggestion
  setAddedFriendList: (friendsID: number[]) => void;
```

Reply:
확장성에도 좋을 것 같네요